patches/chromium/feat_corner_smoothing_css_rule_and_blink_painting.patch

```diff
@@ -0,0 +1,522 @@
+ };
+diff --git a/third_party/blink/public/mojom/renderer_preferences.mojom b/third_party/blink/public/mojom/renderer_preferences.mojom
+index bbcec1dcdaaaf932b3d82c64e8aeb2e7c04b05bf..689205607a763c1d6e040069b1357d84e8ba4bd5 100644
+--- a/third_party/blink/public/mojom/renderer_preferences.mojom
++++ b/third_party/blink/public/mojom/renderer_preferences.mojom
+@@ -201,4 +201,6 @@ struct RendererPreferences {
+   bool uses_platform_autofill = false;
+ 
+   array<uint16> explicitly_allowed_network_ports;
++
++  bool electron_corner_smoothing_css;
+ };
+diff --git a/third_party/blink/public/mojom/use_counter/metrics/css_property_id.mojom b/third_party/blink/public/mojom/use_counter/metrics/css_property_id.mojom
+index ff34492a1b3e0d57847a35f5921337debdf14d10..02e77d7f080de9311f8ac70e3e935dbd38cb285f 100644
+--- a/third_party/blink/public/mojom/use_counter/metrics/css_property_id.mojom
++++ b/third_party/blink/public/mojom/use_counter/metrics/css_property_id.mojom
+@@ -48,6 +48,7 @@ enum CSSSampleId {
+     kInternalForcedVisitedColor = 0,
+     kInternalOverflowBlock = 0,
+     kInternalOverflowInline = 0,
++    kElectronCornerSmoothing = 0,
+ 
+     // This CSSSampleId represents page load for CSS histograms. It is recorded once
+     // per page visit for each CSS histogram being logged on the blink side and the
+diff --git a/third_party/blink/public/web/web_settings.h b/third_party/blink/public/web/web_settings.h
+index 0255879e4ae048554041ca03891fbc7ba8484187..8248d8f54eb5d35caa39a306a6477ace6597b03a 100644
+--- a/third_party/blink/public/web/web_settings.h
++++ b/third_party/blink/public/web/web_settings.h
+@@ -284,6 +284,7 @@ class WebSettings {
+   virtual void SetModalContextMenu(bool) = 0;
+   virtual void SetRequireTransientActivationAndAuthorizationForSubAppsAPIs(
+       bool) = 0;
++  virtual void SetCornerSmoothingCSS(bool) = 0;
+ 
+  protected:
+   ~WebSettings() = default;
+diff --git a/third_party/blink/renderer/build/scripts/core/css/css_properties.py b/third_party/blink/renderer/build/scripts/core/css/css_properties.py
+index 753ba8990f722bafd1770a5e70307cff3764d3f1..16cec517d72887c089f85867e8e37c03199ab394 100755
+--- a/third_party/blink/renderer/build/scripts/core/css/css_properties.py
++++ b/third_party/blink/renderer/build/scripts/core/css/css_properties.py
+@@ -311,7 +311,13 @@ class CSSProperties(object):
+             name_without_leading_dash = property_.name.original
+             if name_without_leading_dash.startswith('-'):
+                 name_without_leading_dash = name_without_leading_dash[1:]
++            # Extra sort level to avoid -internal-* properties being assigned
++            # values too large to fit in a byte.
++            internal_weight = 0
++            if property_.name.original.startswith('-internal'):
++                internal_weight = -1
+             property_.sorting_key = (-property_.priority,
++                                     internal_weight,
```

Comment:
Not a blocker, but could be a good snippet to try upstreaming if it makes sense.

Reply:
Opened [an issue upstream](https://issues.chromium.org/issues/402646511) and it's now [fixed](https://chromium-review.googlesource.com/c/chromium/src/+/6354945), so this part of the patch will be gone in an upcoming upgrade rotation.